src/game-engine/GameEngineA.ts

```diff
@@ -12,18 +11,19 @@ import Accept from './positions/Accept';
 import Reveal from './positions/Reveal';
 import Resting from './positions/Resting';
 import Conclude from './positions/Conclude';
+import { Wallet } from '../wallet';
 
 const fakeGameLibraryAddress = '0xc1912fee45d61c87cc5ea59dae31190fffff232d';
 
 export default class GameEngineA {
   static setupGame({ opponent, stake, balances, wallet }: 
-    { opponent: string, stake: number, balances: number[], wallet: ChannelWallet }
+    { opponent: string, stake: number, balances: number[], wallet: Wallet }
   ) {
     const participants = [wallet.address, opponent];
     const channel = new Channel(fakeGameLibraryAddress, 456, participants);
 
     const nextPledge = new PreFundSetup(channel, 0, balances, 0, stake);
-    const move = wallet.sign(nextPledge.toHex());
+    const move =new Move(nextPledge.toHex(), wallet.sign(nextPledge.toHex()));
```

Comment:
I think we might want `Move` to ultimately be part of the wallet interface. My initial thought was that the wallet would be responsible for taking position-strings from the app and returning `Moves` to be sent to the opponent, and then also for taking `Moves` from the opponent, checking that they're appropriately signed and returning position-strings to the app.

This is fine for now though - we can do all of that as part of a later refactor.